File: CollectEats/CollectEats/Models/Plat.cs
using System;
using System.Collections.Generic;

namespace CollectEats.Models;

public partial class Plat
{
    public int Id { get; set; }

    public string? Nomplat { get; set; }

    public string? Description { get; set; }

    public decimal? Prixbase { get; set; }

    public int? Choixoptionid { get; set; }

    public int? Restaurantid { get; set; }

    public virtual Choixoption? Choixoption { get; set; }

    public virtual Restaurant? Restaurant { get; set; }
}

File: PoCCollectEats/src/app/api/models/plat.ts
/* tslint:disable */
/* eslint-disable */
import { Choixoption } from './choixoption';
import { Restaurant } from './restaurant';
export interface Plat {
  choixoption?: Choixoption;
  choixoptionid?: null | number;
  description?: null | string;
  id?: number;
  nomplat?: null | string;
  prixbase?: null | number;
  restaurant?: Restaurant;
  restaurantid?: null | number;
}
